relayer/relays/beacon/header/syncer/json/beacon_json.go

```diff
@@ -30,27 +31,20 @@ type SyncAggregate struct {
 	SyncCommitteeSignature string `json:"sync_committee_signature"`
 }
 
-type SyncCommitteeUpdate struct {
-	AttestedHeader          BeaconHeader  `json:"attested_header"`
-	NextSyncCommittee       SyncCommittee `json:"next_sync_committee"`
-	NextSyncCommitteeBranch []string      `json:"next_sync_committee_branch"`
-	FinalizedHeader         BeaconHeader  `json:"finalized_header"`
-	FinalityBranch          []string      `json:"finality_branch"`
-	SyncAggregate           SyncAggregate `json:"sync_aggregate"`
-	SyncCommitteePeriod     uint64        `json:"sync_committee_period"`
-	SignatureSlot           uint64        `json:"signature_slot"`
-	BlockRootsRoot          string        `json:"block_roots_root"`
-	BlockRootBranch         []string      `json:"block_roots_branch"`
+type Update struct {
```

Comment:
Since `SyncCommitteeUpdate` is just a `FinalizedUpdate` with one extra field `NextSyncCommittee`,  we merge these two data types together In this PR and frequency is just same as `FinalizedUpdate`.

Reply:
I haven't actually changed the syncing algorithm much at all. Only the data structures and the light client APIs the relayer talks to.

When the relayer needs to submit a new sync committee, it sends this `Update` data structure. And the same for when it only needs to submit a new finalized header.